python/core/qgstaskmanager.sip

```diff
@@ -117,10 +117,10 @@ class QgsTask : QObject
      * Subtasks allow a single task to be created which
      * consists of multiple smaller tasks. Subtasks are not visible or indepedently
      * controllable by users. Ownership of the subtask is transferred.
-     * Subtasks can have an optional list of dependant tasks, which must be completed
+     * Subtasks can have an optional list of dependent tasks, which must be completed
      * before the subtask can begin. By default subtasks are considered independent
      * of the parent task, ie they can be run either before, after, or at the same
-     * time as the parent task. This behaviour can be overriden through the subTaskDependency
+     * time as the parent task. This behaviour can be overridden through the subTaskDependency
```

Comment:
yes there's an impressive list at https://github.com/qgis/qgis3_UIX_discussion/issues/19. If an automatic way can help to fix most of them, it'd be nice.